content/help/ops/perf/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Performance Tuning
+description: Istio performance, sizing and tuning guide.
+weight: 90
+force_inline_toc: true
+---
+## Istio Performance Tuning
+
+Istio's goal is to offer a performance-optimized configuration out of the box.
+However, each use case has different performance criteria. Thus, creating performance-optimized configurations for all use cases is difficult.
+
+The following table shows a few configuration options that affect performance:
+
+| Key | Default Value | Performance Impact | Recommendation |
+| --- | --- | --- | --- |
+| `global.proxy.concurrency` | `2` | An incorrect value can cause high sidecar resource use and latency. | Set this value to equal to the number of CPU cores allocated to the proxy. |
+| `pilot.keepaliveMaxServerConnectionAge` | `30m` | Uneven load distribution across Pilot replicas for a longer period of time. | Reduce this number if you want quicker balancing of load at the cost of increased connect/disconnect activity. |
```

Comment:
I think it is odd for us to have `reduce this number` under recommendation. why would a user wants quicker balancing of load?

Reply:
@linsun Suppose a mesh with many proxies and 1 Pilot instance initially. As the HPA kicks off and new Pilot instances are added, the proxies will continue to be connected to the original one until the connection age expires, maybe overloading a single Pilot. By tuning the connection age, this transition can be quicker, minimizing this situation.